packages/eslint-plugin/src/rules/member-ordering.ts

```diff
@@ -240,30 +250,49 @@ const allMemberTypes = Array.from(
   ).reduce<Set<MemberType>>((all, type) => {
     all.add(type);
 
-    (['public', 'protected', 'private'] as const).forEach(accessibility => {
-      if (type !== 'signature' && type !== 'static-initialization') {
-        all.add(`${accessibility}-${type}`); // e.g. `public-field`
-      }
+    (['public', 'protected', 'private', '#private'] as const).forEach(
+      accessibility => {
+        if (
+          type !== 'signature' &&
+          type !== 'static-initialization' &&
+          type !== 'call-signature' &&
+          !(type === 'constructor' && accessibility === '#private')
+        ) {
+          all.add(`${accessibility}-${type}`); // e.g. `public-field`
+        }
 
-      // Only class instance fields, methods, get and set can have decorators attached to them
-      if (
-        type === 'field' ||
-        type === 'method' ||
-        type === 'get' ||
-        type === 'set'
-      ) {
-        all.add(`${accessibility}-decorated-${type}`);
-        all.add(`decorated-${type}`);
-      }
+        // Only class instance fields, methods, get and set can have decorators attached to them
+        if (
+          accessibility !== '#private' &&
```

Comment:
TIL you can't add decorators to `#private` members.

Reply:
I can create an issue to support decorators from TypeScript 5.